# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/test/unit_ipc/test_alt-data_cross_process_wrap.js
# Commit: 74f99033251c
# Full Hash: 74f99033251cd6c36eb6b0b5a4713eb6b5c793fa
# Author: Jonathan Kingston <jkt@mozilla.com>
# Date: 2019-02-13 09:48:26
# Regressor Bug: 1520868
# File Overlap Count: 1
# Description:
#   Bug 1520868 - Replacing AsyncOpen2 with AsyncOpen always r=valentin
#   
#   Replacing js and text occurences of asyncOpen2
#   Replacing open2 with open
#   
# ==============================================================================

diff -r 3378fd5f4f22 -r 74f99033251c netwerk/test/unit_ipc/test_alt-data_cross_process_wrap.js
--- a/netwerk/test/unit_ipc/test_alt-data_cross_process_wrap.js	Tue Feb 12 17:58:19 2019 +0100
+++ b/netwerk/test/unit_ipc/test_alt-data_cross_process_wrap.js	Tue Feb 12 16:08:25 2019 +0000
@@ -33,7 +33,7 @@
   var chan = make_channel(url);
   var cc = chan.QueryInterface(Ci.nsICacheInfoChannel);
   cc.preferAlternativeDataType("text/binary", "");
-  chan.asyncOpen2(new ChannelListener(readTextData, null));
+  chan.asyncOpen(new ChannelListener(readTextData, null));
 }
 
 function make_channel(url, callback, ctx) {
@@ -65,7 +65,7 @@
   var chan = make_channel(URL);
   var cc = chan.QueryInterface(Ci.nsICacheInfoChannel);
   cc.preferAlternativeDataType("text/parent-binary", "");
-  chan.asyncOpen2(new ChannelListener(readAltData, null));
+  chan.asyncOpen(new ChannelListener(readAltData, null));
 }
 
 function readAltData(request, buffer)